app/Models/Group.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rogue\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Builder;
+
+class Group extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = ['goal', 'group_type_id', 'name'];
+
+    /**
+     * Attributes that can be queried when filtering.
+     *
+     * This array is manually maintained. It does not necessarily mean that
+     * any of these are actual indexes on the database... but they should be!
+     *
+     * @var array
+     */
+    public static $indexes = ['id', 'group_type_id', 'name'];
```

Comment:
I didn't see an `index` added in the migration. Does this still work because it's a `unique` field?

Reply:
The `unique` call in line 22 of the `up` adds a unique index:
```
 $table->unique(['group_type_id', 'name']);
```